lib/mailtrap/errors.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require_relative 'attachment'
+require_relative 'client'
+
+module Mailtrap
+  class AttachmentContentError < StandardError; end
+
+  class Error < StandardError
+    attr_reader :messages
+
+    def initialize(messages)
+      @messages = messages
+
+      super(messages.join(', '))
+    end
+  end
+
+  # AuthorizationError is raised when invalid token is used.
+  class AuthorizationError < Error; end
```

Comment:
How about specifying this inside the error? `#message`?

Reply:
we pass error messages from http response to these classes.
Maybe I would remove the comment because the name is self explanatory
but for now I think it's good as is